src/sentry/static/sentry/app/views/settings/incidentRules/triggers/actionsPanel/index.tsx

```diff
@@ -61,6 +61,8 @@ const getPlaceholderForType = (type: ActionType) => {
     case ActionType.MSTEAMS:
       //no prefixes for msteams
       return 'username or channel';
+    case ActionType.PAGER_DUTY:
```

Comment:
Guessing this should be `ActionType.PAGERDUTY`?

Reply:
It has the underscore [here](https://github.com/getsentry/sentry/blob/master/src/sentry/static/sentry/app/views/settings/incidentRules/triggers/actionsPanel/index.tsx#L33) but I can change that too, I thought there had to be a reason for it.